packages/java/parser-jvm-core/src/main/java/dev/hilla/parser/models/PackageInfoReflectionModel.java

```diff
@@ -23,4 +25,26 @@ public String getName() {
     protected List<AnnotationInfoModel> prepareAnnotations() {
         return processAnnotations(origin.getDeclaredAnnotations());
     }
+
+    @Override
+    public List<PackageInfoModel> getAncestors() {
+        var classLoader = getClass().getClassLoader();
+        return getAllAncestorPackageNames(origin.getName())
+                .map(classLoader::getDefinedPackage).filter(Objects::nonNull)
+                .map(PackageInfoModel::of).toList();
```

Comment:
```suggestion
                .map(PackageInfoModel::of).collect(Collectors::toList);
```
Since it is gonna be cherry-picked to 1.x, we should support Java 11. We could use it in tests but not in modules.